en/developer-documentation.md

```diff
@@ -39,62 +39,62 @@ breadcrumbs:
   </div>
   <div>
     <div>
-      <h2 id="contracts"><img src="/img/icons/ico_contract.svg" class="titleicon" alt="Icon">Contracts</h2>
+      <h2 id="contracts"><span class="fa fa-sitemap fa-lg fa-rotate-270"></span> Contracts</h2>
       <p><a href="/en/developer-guide#contracts">Contracts Guide</a></p>
       <div class="resourcesext">
```

Comment:
@harding class="resourcesext" doesn't serve anything now and could be removed everywhere from the page.